test/WebSites/MvcWithNullable/Program.cs

```diff
@@ -28,7 +29,7 @@
 public class EnumController : ControllerBase
 {
     [HttpGet]
-    public IActionResult Get(LogLevel? logLevel = LogLevel.Error) => Ok(new { logLevel });
+    public Results<Ok<LogLevel?>, NotFound> Get(LogLevel? logLevel = LogLevel.Error) => TypedResults.Ok(logLevel);
```

Comment:
Rather than change an existing method, can you add an extra one?

Also can you add a test for `IResult` via `Results.Ok()` too if that's part of the results support?

Reply:
The IResult does not propagate the typeInfo, I have to look at it.. Because the interface IResult is independent from the IEndpointMetadataProvider :(

Reply:
I think it cannot be fixed from our side